src/common/isolatable_hardwares.cpp

```diff
@@ -207,9 +207,10 @@ std::optional<
     IsolatableHWs::getIsotableHWDetails(
         const IsolatableHWs::HW_Details::HwId& id) const
 {
-    auto it = std::find_if(
-        _isolatableHWsList.begin(), _isolatableHWsList.end(),
-        [&id](const auto& isolatableHw) { return isolatableHw.first == id; });
+    auto it = std::find_if(_isolatableHWsList.begin(), _isolatableHWsList.end(),
```

Comment:
What has been changed here? I can't point a change except clang issue. Run clang format to correct it. Also if anything changed at all then don't take `id` as a reference in lambda rather take it as value as we are only reading it

Reply:
I didn't change anything here, when I ran clang-format it got changed

Reply:
Then run clang in a separate commit. Also why clang would change something which you didn't change?